frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx

```diff
@@ -53,6 +53,13 @@ export const Default: Story = () => (
   </VisualizationWrapper>
 );
 
+// This story has become flaky on CI, so we're skipping it for now.
```

Comment:
This story has been failing in the recent weeks, but I can't track down what's the cause yet. Let's skip it for the time being while we investigate.